### PR TITLE
[ADR] Fix verbosity

### PR DESCRIPTION
**Pull request by haritha-mohan**
Reduce noise during build..

### CODE REVIEW

File: src/Makefile [hunk 1]
```diff
@@ -1224,16 +1224,16 @@ $(DOTNET_DESTDIR)/$($(2)_NUGET_REF_NAME)/ref/$(DOTNET_TFM)/Microsoft.$(1).xml: $
...
 	$$(Q) $$(CP) $($(2)_DOTNET_BUILD_DIR)/ref/Microsoft.$(1).xml $$@
 endif
 endef
 
 build/.build-adr-stamp:
 ifdef ENABLE_XAMARIN
-	$(Q) $(MAKE) build -C $(MACCORE_PATH)/tools/apple-doc-reader
-	$(Q) $(MAKE) create -C $(MACCORE_PATH)/tools/apple-doc-reader
+	$(Q) $(MAKE) build -C $(MACCORE_PATH)/tools/apple-doc-reader 1>/dev/null
+	$(Q) $(MAKE) create -C $(MACCORE_PATH)/tools/apple-doc-reader 1>/dev/null
```
Comment (rolfbjarne) [inline src/Makefile:1236]: I think this solution has a downside: passing V=1 to make (`make V=1`) doesn't make it verbose, it's permanently silenced.

IMHO the best solution would be to copy the `mk/quiet.mk` file to the maccore repository, and then use the variables from that file in `maccore/tools/apple-doc-reader/Makefile`.

So building would be something like:

```make
$(EXEDIR)/AppleDocReader: $(ADR_SOURCES) $(ADR_NUGET_CONFIG)
	$(Q) $(DOTNET) build --configfile $(ADR_NUGET_CONFIG) ./AppleDocReader/AppleDocReader.csproj $(DOTNET_BUILD_VERBOSITY)
``

**Review by rolfbjarne — CHANGES_REQUESTED** [review]

File: src/Makefile (hunk 1 above)
Comment (haritha-mohan) [reply src/Makefile]: @rolfbjarne good idea, made the changes. let me know if it's on par with what you were expecting!

File: mk/xamarin.mk [hunk 2]
```diff
@@ -7,8 +7,8 @@ MONO_BRANCH    := $(shell cd $(MONO_PATH) 2> /dev/null && git symbolic-ref --sho
 endif
 
 ifdef ENABLE_XAMARIN
-NEEDED_MACCORE_VERSION := 054f76b4ec83b17f1fbfdfd14aa5163cb086a69f
-NEEDED_MACCORE_BRANCH := main
+NEEDED_MACCORE_VERSION := 3c4f27a3410d2380d6da0f349eb8ef1cb02e71e5
```
Comment (haritha-mohan) [inline mk/xamarin.mk:10]:
```suggestion
NEEDED_MACCORE_VERSION := 05cc7cbcad56fa018c2bfefd517c2dfc1ba2c711
```

**Review by rolfbjarne — APPROVED** [review]
I think this looks good once the maccore hash/branch has been reverted (this PR doesn't really depend on the maccore bump, so it can be merged independently)

File: mk/xamarin.mk [hunk 3]
```diff
@@ -7,8 +7,8 @@ MONO_BRANCH    := $(shell cd $(MONO_PATH) 2> /dev/null && git symbolic-ref --sho
 endif
 
 ifdef ENABLE_XAMARIN
-NEEDED_MACCORE_VERSION := 054f76b4ec83b17f1fbfdfd14aa5163cb086a69f
-NEEDED_MACCORE_BRANCH := main
+NEEDED_MACCORE_VERSION := 05cc7cbcad56fa018c2bfefd517c2dfc1ba2c711
+NEEDED_MACCORE_BRANCH := fix-verbosity
```
Comment (haritha-mohan) [inline mk/xamarin.mk:11]:
```suggestion
NEEDED_MACCORE_VERSION := f79c721b9be114dfab0cb0e61b23075912168779
NEEDED_MACCORE_BRANCH := main
```

**Review by dalexsoto — APPROVED** [review]

**Pull request merged**